### DISCUSSION TITLE
General Discussion: Feedback, Ideas, and Suggestions for the ARK: Survival Ascended Linux Server Manager

### DISCUSSION THREAD

**Discussion by Zerschranzer** [Zerschranzer/Linux-ASA-Server-Manager#4] [General]
### **Welcome to the general discussion thread for the ARK: Survival Ascended Linux Server Manager!**

This space is dedicated to:
- Sharing your thoughts about the ARK: Survival Ascended Linux Server Manager.
- Suggesting new features or improvements.
- Reporting minor usability issues or quirks.
- Asking general questions about the script or its functionality.

Feel free to:
- Share your experiences using the manager.
- Propose changes or ideas that could make it even better.

Looking forward to hearing your thoughts!

**Comment by mattague** [comment]
I am just jumping into this after hosting a few clusters of ASE for friends in the past. Everything seems so well done, I am genuinely very impressed with the ease of this, especially since it is also managing proton on top of everything. I am extremely grateful for all of the work put into this so far, it has made things a million times easier. I am just thinking of two useful things.

1. I would love a way to sync mods between instances, so I don't have to edit four indivual instance_config.ini files and make sure to add/remove the same number from all of them.

2. Also, would it be possible to copy an existing instance, and just enter the new name, then bring up the instance_config.ini preloaded with the information from another instance, so I don't have to go through and type in the same admin password, server password, max players, and everything else? It could copy the other ini config files into the new instance folder, and warn the user to adjust the map name and save name.


Alternatively, we could have a parent config file that we put some of these values into(modIDs, the first part of the server name(and then in the instance_config.ini append the map name at the end of this variable), max number of players, clusterID, and start up parameters) and have the individiual instances use variables to pull that information from the main file.

**Reply by mattague** [reply]
I was also thinking this could be just for clusters, as it might not be useful otherwise. It would also be handy in the case of clusters to use the same Game.ini and GameUserSettings.ini file between all of the servers, similar to how the ?AltSaveDirectoryName functioned in ASE

**Comment by Kalyandra80** [comment]
Hello
First of all, I would like to say that this script is really great. As a complete newbie in this field, I managed to create a complete cluster using the tool, but I don't know if it's available and I'm just too stupid to find it. It would be nice if the tool would do automatic ARK or mod updates and if you have to shut down the server, it wouldn't shut down immediately, but with a timer and notification that you can set.
Otherwise, the script is really great, especially for beginners.
I apologize for my English, as I have to work with a translator because English is not my native language :) 
Best regards

**Comment by dragginlord-netizen** [comment]
Love the concept (I hate cli). It's so nice that much of the drudgery in managing Ark server(s) is handled through simple easy to use menus. But, right now I can't start a server due to several new bugs. I've tried a few solutions (including updating to a newer Proton version) but no joy. Really don't want to have to go with a plain Ark server Docker container, since that requires so much extra effort. Fingers crossed that the issue will be resolved soonish, in the mean time, back to my Windows server (ugh!).

**Comment by dragginlord-netizen** [comment]
Is there any way to add a Proton configuration to use steamclient.dll instead of steamclient64.dll?  Currently we are having to delete steamclient64.dll to force this, and redelete every time the server is updated. However, it seems that Steam will no longer be supporting the 32 bit client (https://windowsforum.com/threads/valve-moves-steam-windows-client-to-64-bit-only-32-bit-support-ends-2026.391338/), so not sure how this is going to impact us in the future.

**Comment by Spycho2** [comment]
I am having a problem where my gameusersettings.ini files are moving between servers.

I am not sure what the cause is. However, I have my suspicions it is related to the "Start All Servers" command. I havent tested enough to verify this. However, I will fix all the files, turn the servers on 1 by 1. everything looks fine. I check the .ini files. They all look correct. I set up my server restarts to happen while im sleeping. When I wake up the gameusersettings.ini file from 1 server will be in another servers file. and they're all mixed up.

I have no idea what is going on but I have spent hours on this with no success in finding the problem or pinpointing the problem to see if i can work around it..
